test/MultiCompiler.unittest.js

```diff
@@ -1,677 +0,0 @@
-"use strict";
-
-const should = require("should");
```

Comment:
nice to see this converted into a integration test 👍

Reply:
I had to remove them because of the mocking system. Many tests were failing because of additional hooks.